src/lib/Libwin/accesinfo.c

```diff
@@ -311,21 +314,27 @@ secure_file(char *path, char *user, ACCESS_MASK mask)
 static ACL *
 create_secure_dacl2(char *user, ACCESS_MASK mask, char *user2, ACCESS_MASK mask2, SID *owner_sid)
 {
-
-	DWORD					rids[1];
-	gid_t					grp[_MAX_GROUPS];
-	int						i, k;
-	int						cbAcl, cbAce;
-	ACL						*ndacl;
-	SID						*sid;
-	char					*name;
-	char	logb[LOG_BUF_SIZE] = {'\0' } ;
+	DWORD rids[1] = {0};
+	gid_t grp[_MAX_GROUPS] = {0};
+	int i = 0;
+	int k = 0;
+	int cbAcl = 0;
+	int cbAce = 0;
+	ACL *ndacl = NULL;
+	SID *sid = NULL;
+	char *name = NULL;
+	char logb[LOG_BUF_SIZE] = {'\0' };
```

Comment:
Could you fix the indentation?

Reply:
@sid2364 I don't see any indentation issue here.